src/Microsoft.Private.Runtime.UnicodeData/Microsoft.Private.Runtime.UnicodeData.csproj

```diff
@@ -0,0 +1,2 @@
+<Project Sdk="Microsoft.NET.Sdk">
+</Project>

```

Comment:
As this is not a normal TestData project you need to specify these properties manually: https://github.com/dotnet/runtime-assets/blob/ff5202ce3712c3111586d84c1670c947df65ec63/Directory.Build.props#L32-L34.